lib/util/vmdb-logger.rb

```diff
@@ -189,7 +187,7 @@ def call(severity, time, progname, msg, msgnum)
         msg = "#{prefix} #{msg}" unless msg.include?(prefix)
       end
 
-      Format % [msgnum || "----", severity[0..0], format_datetime(time), $$, Thread.current.object_id, severity, progname, msg]
+      Format % [severity[0..0], format_datetime(time), $$, Thread.current.object_id, severity, progname, msg]
```

Comment:
This part might have to stay, that is the `"----"` part, because logger parsers we have rely on it